doc/source/contributing/code_of_conduct_link.md

```diff
@@ -0,0 +1,2 @@
+```{include} ../../../code_of_conduct.md
+```
```

Comment:
What was the reasoning for switching this from an rst to a md file? Does it render differently/better?